docs/topics/Gateway.md

```diff
@@ -763,11 +763,20 @@ A user's presence is their current state on a guild. This event is sent when a u
 
 ###### Game Structure
 
-| Field | Type | Description | Present |
-|-------|------|-------------|---------|
-| name | string | the game's name | Always |
-| type | integer | see [Activity Types](#DOCS_GATEWAY/game-object-activity-types)  | Always |
-| url | ?string | stream url, is validated when type is 1  | When type is 1 |
+| Field | Type | Description |
+|-------|------|-------------|
+| name | string | the game's name |
+| type | integer | [Activity Type](#DOCS_GATEWAY/game-object-activity-types) |
+| url? | ?string | stream url, is validated when type is 1  |
+| timestamps? | [timestamps](#DOCS_GATEWAY/game-object-game-timestamps) object | Unix timestamps for start and/or end of the game |
```

Comment:
nit: all these descriptions are inconsistently formatted with the rest of this document.. they should be all lowercase